lib/questions.js

```diff
@@ -3,7 +3,40 @@
  * @author 唯然<weiran.zsd@outlook.com>
  */
 
-export const langQuestions = [{
+// ------------------------------------------------------------------------------
+// Imports
+// ------------------------------------------------------------------------------
+
+import colors from "ansi-colors";
+
+// ------------------------------------------------------------------------------
+// Helpers
+// ------------------------------------------------------------------------------
+
+/**
+ * Set questions prompt style options in here.
+ * @param {Array} questionsPromptArray Array of questions prompt.
+ * @returns {Array} Questions prompt with style options.
+ */
+function setQuestionsPromptStyle(questionsPromptArray) {
+    return questionsPromptArray.map(opts => ({
+        ...opts,
+        symbols: {
+
+            // For option symbol in select and multiselect
+            indicator: {
+                on: colors.cyan(colors.symbols.radioOn),
+                off: colors.gray(colors.symbols.radioOff)
+            }
+        }
+    }));
+}
+
+// ------------------------------------------------------------------------------
+// Export contents
```

Comment:
```suggestion
// Exports
```

Per our usual convention, I think `Exports` is sufficient here.

https://github.com/eslint/create-config/blob/76fc88595ac475f06004685ee6d2f08e8c245a1c/lib/config-generator.js#L89-L91